site/content/en/docs/drivers/hyperkit.md

```diff
@@ -38,17 +38,3 @@ If you are running other DNS servers, shut them off or specify an alternative bi
 ### Run with logs
 
 Run `minikube start --alsologtostderr -v=7` to debug crashes
-
-### Upgrade HyperKit
-
-New updates to macOS often require an updated hyperkit driver. To upgrade:
-
-* If Docker for Desktop is installed, click on icon in your menu bar and select `Check for updates...`
-* If you have Brew Package Manager, run: `brew upgrade hyperkit`
-* As a final alternative, you install the latest HyperKit from [GitHub](https://github.com/moby/hyperkit/releases)
-* To check your current version, run: `hyperkit -v`
-* If the version didn't change after upgrading verify the correct HyperKit is in the path. run: `which hyperkit`
-
-### Check driver version
-
-Run `docker-machine-driver-hyperkit version` to make sure the version matches minikube
```

Comment:
@afbjorklund was specifially talking about `docker-machine-driver-hyperkit` when he said `minikube now downloads the driver automatically if is missing`.

Everything in this PR minus `### Check driver version` and it's subsection should be reverted.